tests/integration/publish/test_command_integ.py

```diff
@@ -0,0 +1,101 @@
+import re
+import uuid
+import json
+from subprocess import Popen, PIPE
+
+from .publish_app_integ_base import PublishAppIntegBase
+
+
+class TestPublishExistingApp(PublishAppIntegBase):
```

Comment:
All these test require credentials. Travis will fail with no credential error due to the PR not coming from the same repo. To combat this, we only run tests that have credentials when release (until we have a better solution). https://github.com/awslabs/aws-sam-cli/blob/develop/tests/integration/local/invoke/test_integrations_cli.py#L18